dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java

```diff
@@ -242,7 +242,9 @@ public void doSaveProperties(long version) {
             logger.warn("Failed to save registry cache file, will retry, cause: " + e.getMessage(), e);
         } finally {
             if (lockfile != null) {
-                lockfile.delete();
+                if(!lockfile.delete()) {
+                    logger.warn("Failed to delete lock file");
```

Comment:
Add a file name in log message will be better.

Reply:
Good idea!